coralproject/talk
# Markdown Editor Fixes

## What does this PR do?

- Replaced custom loader for markdown editor with `react-loadable`
- Improved introspection result size for embed
- Improved resolve function in development

## How do I test this PR?

1. Run `yarn build`
2. Run `yarn start:development`
3. Go to https://localhost:3000 and login as an admin user
4. Go to configure tab, and check the box for "Ask Readers a Question"
5. See the markdown editor render properly!
